### ISSUE TITLE
Broken link on eth2 merge page

### ISSUE THREAD

**Issue by kwsorensen** [ethereum/ethereum-org-website#3393]
**Describe the bug**

Broken link on the page about the [eth2 merge](https://ethereum.org/en/eth2/merge/).

**To Reproduce**

1. Go to the ethereum page on the [eth2 merge](https://ethereum.org/en/eth2/merge/).
2.  There is a link to [layer 2 scaling solutions](https://ethereum.org/en/developers/docs/layer-2-scaling/) under "The merge and shard chains" header. 
3. That link sends you to a resource not found page.

**Expected behavior**

Link removed or bring you to a valid page.

**Screenshots**

**Desktop (please complete the following information):**

- Browser: Chrome

**Want to contribute?**

I would be up to fix this with a PR as a first issue if we know the correct page (new to the eth community).

**Comment by wackerow** [comment]
Thanks for reporting this @kwsorensen... What's strange is the redirect isn't working correctly from that link.

`netlify.toml` has an automatic redirect:
```
[[redirects]]
  from = "/*/developers/docs/layer-2-scaling"
  to = "/:splat/developers/docs/scaling/layer-2-rollups"
```
but from the link you noted, it seems to go to "Resource not found." Refreshing this link actually then redirects to the new page (for me at least)

We could investigate why the redirect isn't working as expected, but the PR you sent should work in the meantime to get things linked again. See review on that and we'll try to get it live soon =)

**Comment by samajammin** [comment]
This was solved in #3394 - thanks @kwsorensen!
